manifest.json

```diff
@@ -5,15 +5,15 @@
   "description": {
     "en": "Kanboard is a simple visual task board web application"
   },
-  "version": "1.0.48",
+  "version": "1.2.0-0",
   "url": "https://kanboard.net/",
   "license": "AGPL-3.0",
   "requirements": {
 		"yunohost": ">= 2.7.0"
   },
   "maintainer": {
-    "name": "mbugeia",
-    "email": "maxime@max.privy.place"
+    "name": "jibec",
```

Comment:
Why remove mbugeia ?

Reply:
In fact, I know the previous discussion was transfered on dev.yunohost.org, but I'm still against removing previous maintainer... for me this PR is not ok.

Reply:
Well, because I rewrote the package and wish to assume the maintenance of the app

Reply:
see here : https://github.com/YunoHost-Apps/umap_ynh/blob/master/manifest.json#L14
2 maintainers.

Reply:
well, yes, I agree with you. You are the maintainer now, but you still use work of mbugeia ? so, I think you need to keep his name, at least somewhere in the README, or we will wait for a solution (through the issue).

Reply:
We already discussed this in another package, I don't really remember what was the result, I think the manifest were going to evolve.
I also remember we discussed this on the app xmpp room before making the change. We are talking about something 4 month old already...

Maybe @maniackcrudelis can tell us the good practice here, I don't want to obfuscate the existence or the work of previous maintainer, but don't see the point to let inactive point of contact.

Reply:
Exactly, we have already told about that matter in another issue (I don't remember which one though).
The final answer was to add a field in the manifest for previous maintainers.
A additionnal field will not bother YunoHost, and we can keep the current maintainer as the real person to contact in need of. But keep all the previous maintainer in the manifest and like that respect their previous work.

Reply:
yes : this issue : https://github.com/YunoHost-Apps/kanboard_ynh/pull/55... but to be honest I'm not really convinced by what was said...

Reply:
Do you have another idea or a reason to share against that idea ?

Reply:
So, Jibec, can you add this entry in the manifest please ?